### PR TITLE
Bug 1934329: Bump sanitize-html version to 2.3.2 to improperly validate the hostnames

### PR DESCRIPTION
**Pull request by jhadvig**
Improper validation of hostnames set by the "allowedIframeHostnames" option can lead to bypass hostname whitelist for iframe element. Fixing this even though we are not using this option in this repo.

/assign @spadgett

### CODE REVIEW

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
Holding until #8289 merges to unblock CI.

/hold

**Review by zherman0 — APPROVED** [review]

**Pull request merged**
